lib/rubocop/cop/rspec/cop.rb

```diff
@@ -58,7 +65,17 @@ def relevant_rubocop_rspec_file?(file)
         end
 
         def rspec_pattern
-          Regexp.union(rspec_pattern_config.map(&Regexp.public_method(:new)))
+          if rspec_config_defined?
```

Comment:
Currently the only `RSpec` configuration is `Patterns`, but that may not be the case in the future. Perhaps it would be better to check for `rspec_pattern_config_defined?` – e.g. 
```rb
def rspec_pattern_config_defined?
  rspec_config_defined? &&
    config
      .for_all_cops
      .fetch('RSpec')
      .key?('Patterns')
end
```
What do you think?

Reply:
OK. I'll change to check it.

Reply:
@bquorning updated